library/core/class.controller.php

```diff
@@ -621,6 +621,13 @@ public function definitionList($wrap = true) {
             $this->_Definitions['Search'] = t('Search');
         }
 
+        if (!array_key_exists('Roles', $this->_Definitions)) {
```

Comment:
this is not an appropriate place for this, because the controller is part of the framework, while role data is part of the Dashboard app.